trillian_map_api.proto

```diff
@@ -111,4 +118,5 @@ service TrillianMap {
         get: "/v1beta1/maps/{map_id}/roots/{revision}"
       };
   }
+  rpc InitMap(InitMapRequest) returns(InitMapResponse) {}
```

Comment:
Maps are initialized during `CreateTree` already. 
Could you expand on the use case in mind?

Reply:
That is not true :)
The Map is created by CreateTree, but the map code fails spectacularly if there isn't an initial rev 0 Map head, which is why there were a bunch of calls to `Init()` dotted throughout the RPC server inline with all the incoming requests.  Unfortunately, this leads to multiplication of the number of in-flight transactions, and also makes it impossible (or very hard) to refactor the storage implementation to support other transaction programmatic models (see #954).